app/scrapers/gigablast.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import print_function
+from generalized import Scraper
+
+
+class Yandex(Scraper):
+    """Scrapper class for Gigablast"""
+
+    def __init__(self):
+        self.url = 'https://www.gigablast.com/search'
+        self.defaultStart = 0
+
+    def parseResponse(self, soup):
+        """ Parse the response and return set of urls
+        Returns: urls (list)
+                [[Tile1,url1], [Title2, url2],..]
+        """
+        urls = []
+        for a in soup.findAll('a', {'class': 'link link_theme_normal'}):
```

Comment:
wrong please do -inspect element for anchors at http://www.gigablast.com/search?c=main&q=fossasia

Reply:
i didnt get what you are saying. can you please explain

Reply:
do you know how to inspect element in any webpage?

Reply:
no. how are we supposed to do that?

Reply:
Check out some tutorials on web scraping

Reply:
@bhaveshAn  what are anchors?

Reply:
@starlord1311 Anchors are the basic part of web development, if you don't know these, then please start learning web development and please come to contribute only after learning these.
Also none of your PR is working please give some relevant code contributions otherwise some other person should work on these issues.
Both your PRs are open from about 28 days. Submit any of these PRs with commitment of success in 2 days.
Otherwise close these if you can't work.

Reply:
@bhaveshAn i did learn about web dev. the latest pr is working only the icon cant be seen.

Reply:
@bhaveshAn codacy did approve but idk why travis is not approving.

Reply:
Travis has hunged. May be resolving conflicts can solve the problems